dht.go

```diff
@@ -249,32 +254,15 @@ func makeDHT(ctx context.Context, h host.Host, cfg config) (*IpfsDHT, error) {
 }
 
 func makeRoutingTable(dht *IpfsDHT, cfg config) (*kb.RoutingTable, error) {
-	self := kb.ConvertPeerID(dht.host.ID())
-	// construct the routing table with a peer validation function
-	pvF := func(c context.Context, p peer.ID) bool {
-		// connect should work
-		if err := dht.host.Connect(c, peer.AddrInfo{ID: p}); err != nil {
-			rtPvLogger.Infof("failed to connect to peer %s for validation, err=%s", p, err)
-			return false
-		}
+	// To grok the Math Wizardy here, please be patient as a document explaining it will
+	// be published soon.
```

Comment:
I'd put in some line here like "The threshold is calculated based on the expected amount of time that should pass before we query a peer as part of our refresh cycle." and then add the "please be patient line"

Reply:
Done.